corehq/ex-submodules/dimagi/utils/post.py

```diff
@@ -35,77 +26,3 @@ def simple_post(data, url, content_type="text/xml", timeout=60, headers=None, au
         kwargs["verify"] = verify
 
     return requests.post(url, data, **kwargs)
-
-
-def post_data(data, url, curl_command="curl", use_curl=False,
-              content_type="text/xml", path=None, use_chunked=False,
-              is_odk=False, attachments=None):
-    """
-    Do a POST of data with some options.  Returns a tuple of the response
-    from the server and any errors
-
-    if it's ODK, then also process any additional attachments that are an array
-    of tuples of the name and the path
-
-    """
-    attachments = attachments or []
-    results = None
-    errors = None
-
-    if path is not None:
-        with open(path, 'rb') as f:
-            data = f.read()
-
-    try:
-        if use_curl:
-            if path is None:
-                tmp_file, path = tmpfile()
-                with tmp_file:
-                    tmp_file.write(data)
-            params = [curl_command, '--request', 'POST' ]
-            params.append('--insecure')
-            if is_odk == False:
-                #it's legacy j2me
-                params.append('--header')
-                params.append('Content-type:%s' % content_type)
-                params.append('--data-binary')
-                params.append('@%s' % path)
-            else:
-                params.append('-F')
-                params.append('xml_submission_file=@%s' % path)
-
-
-                if attachments:
-                    for attach in attachments:
-                        params.append('-F')
-                        params.append('%s=@%s' % (attach[0], attach[1]))
-
-            if use_chunked:
-                params.append('--header')
-                params.append('Transfer-encoding:chunked')
-            else:
-                if not is_odk:
-                    params.append('--header')
-                    params.append('Content-length:%s' % len(data))
-
-            params.append(url)
-            p = subprocess.Popen(params,
-                                 stdout=PIPE, stderr=PIPE, shell=False)
-            errors = p.stderr.read()
-            results = p.stdout.read()
-        else:
-            results = simple_post(data, url, content_type).read()
-            
-    except Exception as e:
-        errors = str(e)
-
-    return results, errors
-
-
-def post_file(filename, url, curl_command="curl", use_curl=False,
```

Comment:
note: this is method is not used anywhere and `post_data` is called from within this only.